ledger/src/blockstore_db.rs

```diff
@@ -1995,13 +1998,26 @@ pub struct LedgerColumnOptions {
     // Determine the way to compress column families which are eligible for
     // compression.
     pub compression_type: BlockstoreCompressionType,
+
+    // Control the number of performance samples in a million for RocksDB reads
+    // and writes.
+    pub perf_context_samples_in_1m: u32,
+
+    // A counter to determine whether to sample the current RocksDB read operation.
+    pub perf_read_counter: Arc<AtomicU32>,
+
+    // A counter to determine whether to sample the current RocksDB write operation.
+    pub perf_write_counter: Arc<AtomicU32>,
 }
 
 impl Default for LedgerColumnOptions {
     fn default() -> Self {
         Self {
             shred_storage_type: ShredStorageType::RocksLevel,
             compression_type: BlockstoreCompressionType::default(),
+            perf_context_samples_in_1m: 0,
+            perf_read_counter: Arc::new(AtomicU32::new(0)),
+            perf_write_counter: Arc::new(AtomicU32::new(0)),
```

Comment:
If you are implementing the `default` method, 
then probably `Arc::<AtomicU32>::default()` is a more explicit syntax.

Reply:
Good to know this.  Will give it a try!  Hopefully this will also solve the private field issue.